src/Containers/Microsoft.NET.Build.Containers/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -77,6 +77,10 @@ Microsoft.NET.Build.Containers.Tasks.CreateNewImage.RuntimeIdentifierGraphPath.g
 Microsoft.NET.Build.Containers.Tasks.CreateNewImage.RuntimeIdentifierGraphPath.set -> void
```

Comment:
Unrelated to this PR, have we missed moving parts of the API from Unshipped to Shipped? This seems like a lot of additions for 3xx.

Reply:
Correct - we have never moved these over to Shipped.

Reply:
However, we don't publish this package in a way that's actually consumable - it's not in `lib` package folders so no one taking a nuget package would ever consume these libraries.

Reply:
I'm satisfied with the move to internal I think, though if I forget and comment this again in 6 months we should maybe change it so I stop getting distracted.